pytorch_lightning/trainer/training_loop.py

```diff
@@ -490,7 +490,8 @@ def optimizer_step(self, optimizer, opt_idx, batch_idx, train_step_and_backward_
                 ' To request, please file a Github issue in PyTorch and tag @mcarilli')
 
         # wraps into LightingOptimizer only for running step
-        optimizer = LightningOptimizer.to_lightning_optimizer(optimizer, self.trainer)
+        optimizer = LightningOptimizer.to_lightning_optimizer(
+            optimizer, opt_idx, self.trainer)
```

Comment:
```suggestion
        optimizer = LightningOptimizer.to_lightning_optimizer(optimizer, opt_idx, self.trainer)
```
what is your line length? it should be 120

Reply:
done